src/Type/Definition/ObjectType.php

```diff
@@ -147,7 +150,7 @@ public function hasField($name)
     public function getFields()
     {
         if ($this->fields === null) {
-            $fields       = $this->config['fields'] ?? [];
+            $fields       = $this->config['fields'] ?: [];
```

Comment:
The short ternary operator is not strict and disallowed by CS, can you mb rebase onto master to see if it gets caught?

https://github.com/doctrine/coding-standard/pull/57

Reply:
https://github.com/webonyx/graphql-php/pull/645 :)